src/Compression/CompressionCodecT64.cpp

```diff
@@ -500,14 +499,21 @@ UInt32 compressData(const char * src, UInt32 bytes_size, char * dst)
     static constexpr const UInt32 matrix_size = 64;
     static constexpr const UInt32 header_size = 2 * sizeof(UInt64);
 
-    if (bytes_size % sizeof(T))
-        throw Exception(ErrorCodes::CANNOT_COMPRESS, "Cannot compress with T64 codec, data size {} is not multiplier of {}",
-                        bytes_size, sizeof(T));
+    UInt8 bytes_to_skip = bytes_size % sizeof(T);
+    bytes_size -= bytes_to_skip;
+    memcpy(dst, src, bytes_to_skip);
```

Comment:
Is it a forward-backward incompatible change in codec? Could older CH decompress the data compressed by newer `compressData` and vice versa?

Reply:
It does not look backward compatible (old version won't be able to read data compressed by a newer version), but should be forward compatible (new version will be able to read data compressed by an old version).